region/lowernorfair/east.json

```diff
@@ -2702,51 +2708,136 @@
           "to": [
             {
-              "id": 4,
+              "id": 6,
               "requires": [
-                "heatProof",
-                "CanTrickyUseFrozenEnemies",
-                "canCeilingClip",
+                {"or":[
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"cost":[ {"heatFrames": 80}]},
+                {"or":[
+                  {"and":[
+                    {"canComeInCharged": {
+                      "fromNode": 1,
+                      "framesRemaining": 180,
+                      "shinesparkFrames": 0
+                    }},
+                    {"cost":[ {"heatFrames": 40}]}
+                  ]},
+                  {"canComeInCharged": {
+                    "fromNode": 1,
+                    "framesRemaining": 0,
+                    "shinesparkFrames": 80
+                  }}
+                ]},
+                {"obstacle":{
+                  "id": "E",
+                  "requires": null
+                }},
                 {"obstacle":{
                   "id": "C",
-                  "requires": [
-                    {"or": [
-                      "Plasma",
-                      "Missile",
-                      "Super",
-                      "ScrewAttack"
-                    ]}
-                  ]
+                  "requires": null
                 }},
                 {"obstacle":{
                   "id": "F",
-                  "requires": [
-                    {"or": [
-                      "Plasma",
-                      "Missile",
-                      "Super",
-                      "ScrewAttack"
-                    ]}
-                  ]
-                }},
+                  "requires": null
+                }}
+              ],
+              "unlock": null
+            },
+            {
+              "id": 7,
+              "requires": [
+                {"or":[
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"cost": [{"heatFrames": 200}]},
+                {"or":[
+                  {"obstacle":{
+                    "id": "E",
+                    "requires": ["never"]
+                  }},
+                  {"cost":[
+                    {"enemyDamage": [{
+                      "enemy": "Multiviola",
+                      "type": "contact",
+                      "hits": 1
+                    }]}
+                  ]}
+                ]},
+                {"or":[
+                  {"obstacle":{
+                    "id": "C",
+                    "requires": ["never"]
+                  }},
+                  {"cost":[
+                    {"enemyDamage": [{
+                      "enemy": "Dessgeega",
+                      "type": "contact",
+                      "hits": 1
+                    }]}
+                  ]}
+                ]},
+                {"and":[
+                  {"obstacle":{
+                    "id": "C",
```

Comment:
Is this thing with requiring obstacle C & E multiple times in the same req really going to work without triggering the contact damage options?

Reply:
It's not going to work, no. The Dessgeega damage isn't in the right place. An `or` is missing. And there seems to be a pointless `and` in there somewhere too.
It's supposed to be:
- heatproof or canHeatRun
- Take damage from Multiviola if it wasn't previously killed
- Take an additional hit from a Dessgeega, OR kill the Dessgeegas. OR kill both the Dessgeegas and Multiviolas